spec/cc/engine/analyzers/sexp_lines_spec.rb

```diff
@@ -49,8 +49,8 @@ module CC::Engine::Analyzers
 
         expect(locations.count).to eq 2
 
-        expect([locations[0].begin_line, locations[0].end_line]).to eq([2, 7])
-        expect([locations[1].begin_line, locations[1].end_line]).to eq([11, 16])
+        expect([locations[0].begin_line, locations[0].end_line]).to eq([1, 7])
+        expect([locations[1].begin_line, locations[1].end_line]).to eq([10, 16])
```

Comment:
Probably fixes to ruby_parser. I have a release coming out soon that makes more of these fixes so there might be more failed tests coming up.